src/tools/general.js

```diff
@@ -252,6 +248,26 @@ function getRequestListSources({ locationId, includeHotels = true, includeRestau
  * @typedef {UnwrappedPromiseFn<typeof getClient>} Client
  */
 
+/**
+ * @param {Apify.Session} [session]
+ */
+async function getApiClient(session) {
```

Comment:
Could this be as part of the crawler?

Reply:
Is not being used anymore, I will remove it